src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -3944,13 +3944,53 @@ public void NullCoalesingAssigmentExpressionAndCoalescingOperator()
             EOF();
         }
 
+        [Fact]
+        public void NullCoalescingAssignmentExpressionNested()
+        {
+            UsingExpression("a ??= b ??= c");
```

Comment:
>a ??= b ??= c [](start = 29, length = 13)

Should we test `(a ??= b) ??= c` as well?

Reply:
We can test it from a syntax perspective, but that's not semantically valid as `a ??= b` can't be an lvalue.

---
In reply to: [216108148](https://github.com/dotnet/roslyn/pull/29691#discussion_r216108148) [](ancestors = 216108148)